### PR TITLE
Remove programmer/date from comments

### PR DESCRIPTION
**Pull request by derobins**
* Removes Programmer: and Date: fields
* Fixes a few Modifications: fields leftover from previous work

### CODE REVIEW

**Comment by derobins** [comment]
No code changes. Comments only. Apologies for the size, but this should be quick to scan.

File: bin/h5vers [hunk 1]
```diff
@@ -16,8 +16,6 @@ use strict;
 # If you do not have access to either file, you may request a copy from
 # help@hdfgroup.org.
 #
```
Comment (hyoklee) [inline bin/h5vers:18]: Remove line 18.

File: bin/make_err [hunk 2]
```diff
@@ -20,8 +20,6 @@ use warnings;
 # Read in the error description text file and create the appropriate headers
 # needed by the library.
 #
```
Comment (hyoklee) [inline bin/make_err:22]: Remove line 22.

**Review by hyoklee — APPROVED** [review]
It would be nice if you could remove empty comment lines that start with `#`.

**Comment by derobins** [comment]
> It would be nice if you could remove empty comment lines that start with `#`.

The library code frequently adds an extra '#' to comment blocks in shell scripts. I made no attempt to make the comments uniform in this regard (they differ wildly in form as it is). I plan to do further cleaning of comments (yay, sed) in the future.

**Review by byrnHDF — APPROVED** [review]

**Pull request merged**